source/core/ExecutionContext.cpp

```diff
@@ -436,7 +436,7 @@ void ExecutionContext::IsrEnqueue(QueueElt* elt)
 DEFINE_VIREO_BEGIN(Execution)
     DEFINE_VIREO_REQUIRE(VirtualInstrument)
     DEFINE_VIREO_FUNCTION(FPSync, "p(i(String))")
-    DEFINE_VIREO_FUNCTION(CurrentBrowserFPS, "p(o(Double) io(" ERROR_CLUST_TYPE_STRING "))")
+    DEFINE_VIREO_FUNCTION(CurrentBrowserFPS, "p(o(Double) io(ErrorClust))")
```

Comment:
OK, I renamed it.  @cglzaguilar Please make sure to update DFIR if you are generating refs to this type.  I'm not sure if you are yet; all I've seen is replications of the whole cluster type.